app/ghostery-browser-hub/Views/OnboardingView/OnboardingConstants.js

```diff
@@ -0,0 +1,16 @@
+/**
+Constants for Ghostery Browser Intro Hub Onboarding
+These constants map the onboarding step numbers to more memorable names
+ */
```

Comment:
I think this might be overriding an updated header, could you double check? (Also possible that this branch is a couple commits behind onboarding)

Reply:
Merged and updated